src/Spago/Prelude.purs

```diff
@@ -159,7 +159,7 @@ mkTemp' maybeSuffix = liftAff do
     sha <- Sha256.hashString $ show now <> fromMaybe "" maybeSuffix
     shaToHex sha
   -- Return the dir, but don't make it - that's the responsibility of the client
-  let tempDirPath = Path.concat [ Paths.paths.temp, random ]
+  let tempDirPath = Path.concat [ Paths.paths.temp, String.drop 50 random ]
```

Comment:
I had to make the temp directory names shorter, because Windows has a path length limit, and nesting a git repository within the temp directory was getting way over. This is obviously not quite sustainable. I would love to replace this with GUIDs, but that would require another NPM dependency.

Reply:
Yeah this should be fine!